CHANGELOG.md

```diff
@@ -13,10 +13,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   by adding a class-level variable called `authorized_missing_keys` to any PyTorch module that a `Model` uses.
   If defined, `authorized_missing_keys` should be a list of regex string patterns.
 - Added `FBetaMultiLabelMeasure`, a multi-label Fbeta metric. This is a subclass of the existing `FBetaMeasure`.
+- A `MultiTaskModel` and abstractions to use with it, including `Backbone` and `Head`.  The
```

Comment:
Note: the `vision` branch should probably have a separate `unreleased-2.0` or something, as this is currently mixing things that will get released from the master branch before 2.0 with 2.0 stuff.  I moved the 2.0 stuff I recognized up from the 1.1 release notes, but it'll just get stale again with the next release.

Reply:
@dirkgr, @epwalsh, I just updated the changelog to fix this; it might be good to double check the things I moved to make sure I didn't miss anything.  Feel free to change how I did this if you want to separate these things differently.